### ISSUE TITLE
The value of lat is different from total_loss - ce

### ISSUE THREAD

**Issue by JunrQ** [JunrQ/NAS#4]
@JunrQ 
put `lat` in `loss = [..., stop_grad(lat)]`
and print it with `print(outputs[-1].asnumpy())`
